packages/block-editor/src/components/use-block-commands/index.js

```diff
@@ -127,59 +126,6 @@ export const useTransformCommands = () => {
 	return { isLoading: false, commands };
 };
 
-const useActionsCommands = () => {
-	const { clientIds } = useSelect( ( select ) => {
-		const { getSelectedBlockClientIds } = select( blockEditorStore );
-		const selectedBlockClientIds = getSelectedBlockClientIds();
-
-		return {
-			clientIds: selectedBlockClientIds,
-		};
-	}, [] );
-
-	const { getBlockRootClientId, canMoveBlocks, getBlockCount } =
-		useSelect( blockEditorStore );
-
-	const { setBlockMovingClientId, setNavigationMode, selectBlock } =
-		useDispatch( blockEditorStore );
-
-	if ( ! clientIds || clientIds.length < 1 ) {
-		return { isLoading: false, commands: [] };
-	}
-
-	const rootClientId = getBlockRootClientId( clientIds[ 0 ] );
-
-	const canMove =
-		canMoveBlocks( clientIds ) && getBlockCount( rootClientId ) !== 1;
-
-	const commands = [];
-
-	if ( canMove ) {
-		commands.push( {
-			name: 'move-to',
-			label: __( 'Move to' ),
```

Comment:
Why are we removing the move to command? This is the only way to move blocks via keyboard, how is this incumbering edit mode?

Reply:
I think it was mentioned in the PR description. Move To utilized the selection mode, so with that now gone, it ends up broken. I guess it'll need to be replaced with something else. We'd need to make an issue if this PR is merged.

Reply:
Oh I missed that. It seems a bit of an over extension though. If this is the case than we should treat this PR as an exploration. If we merge it we remove an unrelated affordance in trunk with no alternative nor a plan to fix it.

Reply:
The reasoning is that very few people actually use that feature and for most user testing that I saw, it actually creates confusion and issues for users more than it solves anything. Users wonder why nothing happens when they click somewhere to move the block, they don't really know that they should be using keyboard. I think the feature need to be rethought entirely. It's a good idea but we didn't come up with the right approach there anyway.